tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/DdwafRuleSetInfoStruct.cs

```diff
@@ -0,0 +1,34 @@
+// <copyright file="DdwafRuleSetInfoStruct.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.AppSec.Waf.NativeBindings
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct DdwafRuleSetInfoStruct
+    {
+        /// <summary>
+        /// Number of rules successfully loaded
+        /// </summary>
+        public ushort Loaded;
+
+        /// <summary>
+        /// Number of rules which failed to parse
+        /// </summary>
+        public ushort Failed;
+
+        /// <summary>
+        ///  Map from an error string to an array of all the rule ids for which that error was raised. { error: [rule_ids]} **/                                                                                                                                                                                                                                                                    that error was raised. { error: [rule_ids]}
```

Comment:
I think there may be a copy/paste error on this line

```suggestion
        ///  Map from an error string to an array of all the rule ids for which that error was raised. { error: [rule_ids]}
```

Reply:
thanks fixed in 50c1359ab57b5cc2a9503358084816b10d950f65